streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java

```diff
@@ -280,30 +296,37 @@ private void assertTopologyContainsProcessor(final Topology topology, final Stri
             }
         }
         throw new AssertionError("No processor named '" + processorName + "'"
-                + "found in the provided Topology:\n" + topology.describe());
+            + "found in the provided Topology:\n" + topology.describe());
     }
 
     @Test
-    public void shouldCreateSourceAndSinkNodesForRepartitioningTopic() throws NoSuchFieldException, IllegalAccessException {
+    public void shouldCreateSourceAndSinkNodesForRepartitioningTopic() throws Exception {
```

Comment:
Relaxing the throws declaration tightens up the line, but does it make the test more mysterious?

Reply:
No (compare my comment on the other PR). If any exception occurs, the test fails anyway.

Reply:
I buy it. Thanks!